app/move/controllers/timeline.js

```diff
@@ -0,0 +1,187 @@
+const pluralize = require('pluralize')
+
+const { event: eventModel } = require('../../../common/lib/api-client/models')
+const populateResources = require('../../../common/lib/populate-resources')
+const presenters = require('../../../common/presenters')
+const moveService = require('../../../common/services/move')
+const i18n = require('../../../config/i18n')
+
+const getTabsUrls = require('./view/view.tabs.urls')
+
+const addMockValuesToEvents = (req, moveEvents) => {
```

Comment:
Horrible. Should probably be removed before merging. Or enabling only if `API_USE_MOCKS` is on